euth/dashboard/forms.py

```diff
@@ -115,6 +115,9 @@ class Meta:
         model = project_models.Project
         fields = ['name', 'description', 'image', 'information', 'is_public',
                   'result']
+        widgets = {'is_public': RadioSelect(choices=[
+                (True, _('Private')),
+                (False, _('Not Private'))])}
```

Comment:
'Not Private' should be 'Public', and the booleans should be the other way round. (As the model field is called is_public, the field should be set true, when the project is public and false, when it is private.)